### DISCUSSION TITLE
v1.34.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1313] [General]
## Features
- You can now view screen sizes in the more details section
![image](https://user-images.githubusercontent.com/477975/180183339-7bb29858-b154-4468-9d1b-9540fd566790.png)

- Just like the custom tracking script name, you can now customize the `/api/collect` endpoint to avoid some ad-blockers. Just set the environment variable `COLLECT_API_ENDPOINT` to a different location:
```
COLLECT_API_ENDPOINT=/api/x
```

## Updates 
- Fixed some Unicode display issues
- Default MySQL to InnoDB
- Language translation updates
- Performance improvements for some database queries
- Upgraded Next.js to `12.2.0` and Prisma to `4.1.0`.

A huge thanks to all the contributors on this release! @MarcHagen @briancao @rohandebsarkar @njfdev @fredrikj31 @thueske @sifferhans @liaohongxing 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.34.0'>v1.34.0</a>.</em>

**Comment by AliMickey** [comment]
Waiting for docker release

**Reply by twiesing** [reply]
See: https://github.com/umami-software/umami/pull/1319

**Reply by AliMickey** [reply]
Thanks, updated fine now. 

However the custom api endpoint url is not working. Is there a template to follow?

This is mine now however it still tries to POST to /api/collect

`- COLLECT_API_ENDPOINT=/api/mickit`

**Reply by mikecao** [reply]
COLLECT_API_ENDPOINT is a build time variable. If you're using docker you'll need to rebuild the image.

**Reply by AliMickey** [reply]
I stopped both umami and the db containers, removed them, deleted the images. Did a compose up, disabled cache in my browser and it's still hitting /api/collect

**Reply by mikecao** [reply]
You're right, it's because the `build-tracker` script didn't import the environment variables. I'll have a fix.

**Reply by ghost** [reply]
Yes, there must be some problem because I have done a lot of tests and I cannot change the COLLECT_API_ENDPOINT.

**Reply by mikecao** [reply]
Fix is in `dev` branch if you want to test it early.
